README.md

```diff
@@ -1,24 +1,24 @@
 # flaxvision
 The flaxvision package contains a selection of neural network models ported from [torchvision](https://github.com/pytorch/vision) to be used with [JAX](https://github.com/google/jax) & [Flax](https://github.com/google/flax).
 
-**Note: flaxvision is currently in active development. API and functionality will change before the first stable release.**
+**Note: flaxvision is currently in active development. API and functionality may change between releases.**
 
-### Roadmap to V0.1
+### Roadmap to v0.1.0
 Planned features for the first release:
-- [ ] Update models to [linen API](https://github.com/google/flax/tree/0132b3f234a9868b47df491efde870bdc58e97a9/linen_examples)
-- [ ] Add support for transfer learning and checkpoint loading (with examples)
-- [ ] Add support to ResNet for dilated convolutions
-- [ ] Port DeepLabv3 model for image segmentation
+- [x] Update models to [linen API](https://github.com/google/flax/tree/0132b3f234a9868b47df491efde870bdc58e97a9/linen_examples)
+- [x] Add support for transfer learning
+- [x] Add support to ResNet for dilated convolutions
+- [x] Port DeepLabv3 model for image segmentation
 
 ## Quickstart
-### Example
+### Transfer Learning Example
 ```python
 from jax import random
 from flaxvision import models
 
 rng = random.PRNGKey(0)
 
-model = models.vgg16(rng, pretrained=True)
+pretrained_model = models.vgg16(rng, pretrained=True)
```

Comment:
Should this example show an actual fine-tuning loop?

Reply:
Yes, finishing up on a detailed colab these days. Will update the readme too!